openedx/features/learner_profile/static/learner_profile/js/learner_profile_factory.js

```diff
@@ -139,7 +145,10 @@
                     valueAttribute: 'bio',
                     helpMessage: '',
                     persistChanges: true,
-                    messagePosition: 'header'
+                    messagePosition: 'header',
+                    maxCharacters: 300,
+                    charCountMessage: gettext('characters remaining'),
```

Comment:
Do we care about singular for 1? We could either put the s in parentheses, or use the i18n helper for plurals.